### PR TITLE
first draft of Cinder multi attach

### PR DESCRIPTION
**Pull request by lvarin**
## Proposed changes

Briefly describe the changes you've made here. Remember to add a link to the [preview page](https://csc-guide-preview.rahtiapp.fi/origin/cinder-multiattach/) of your branch.

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes all tests.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/cinder-multiattach/) (select your branch from the list).

### CODE REVIEW

File: docs/cloud/pouta/multiattach.md [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+### CLI
+
+Before doing this, you need to [install the openstack client](../install-client/)):
+
+1. Create a multi attach volume:
+
+    ```sh
+    openstack volume create --size <size_in_GB> --type multiattach <volume_name>
+    ```
+    You need to replace `<volume_name>` by the name iyou want to give to the volume, and the `<size_in_GB>` by the size in Gigabytes you want the volume to have.
```
Comment (trispera) [inline docs/cloud/pouta/multiattach.md:40]: A small typo. `iyou` > `you`

File: docs/cloud/pouta/multiattach.md [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+
+    ```sh
+    sudo dnf config-manager --enable  highavailability resilientstorage
+    sudo dnf install pacemaker corosync pcs dlm gfs2-utils
+    ```
+
+    1. For Ubuntu and other Debian based distributions you just need to install 3 packages:
+
+    ```sh
+    apt install gfs2-utils dlm-controld linux-modules-extra-<ansible_kernel>
```
Comment (trispera) [inline docs/cloud/pouta/multiattach.md:136]: I don't really understand the meaning of `ansible` in `<ansible_kernel>`. Did you mean ubuntu kernel? If so, maybe this command could replace:
```sh
apt install gfs2-utils dlm-controld linux-modules-extra-$(uname -r)
```

File: docs/cloud/pouta/multiattach.md [hunk 3]
```diff
@@ -0,0 +1,452 @@
...
+In order to install OCFS2, you need to follow few steps:
+
+1. Install the VM nodes. There is no special consideration on this step, other than making sure the nodes can see each other in the Network (it is the default behaviour of VM nodes created in the same Pouta project), and that they are installed with the same distribution version. We have tested this with `Ubuntu v22.04` and `AlmaLinux-9`, other distributions and versions might also work, but we have not tested them. This guide will use Ubuntu as an example. AlmaLinux requires to install an specific Oracle kernel.
+
+1. Create and attach the volume. See the manual [Create and attach a volume](#create-and-attach-a-volume) from above.
+
+1. Install the OCFS2 software:
+
+    ```sh
+    ocfs2-tools linux-modules-extra-<kernel_version> linux-image-<kernel_version>
```
Comment (trispera) [inline docs/cloud/pouta/multiattach.md:354]:
```
ocfs2-tools linux-modules-extra-<kernel_version> linux-image-<kernel_version>
```
by
```
ocfs2-tools linux-modules-extra-<kernel_version> linux-image-$(uname -r)
```
?

Comment (lvarin) [reply docs/cloud/pouta/multiattach.md]: fixed

File: docs/cloud/pouta/multiattach.md (hunk 1 above)
Comment (lvarin) [reply docs/cloud/pouta/multiattach.md]: fixed

File: docs/cloud/pouta/multiattach.md [hunk 4]
```diff
@@ -0,0 +1,499 @@
...
+       other nodes.
+
+    norecovery
+       A synonym for spectator
+    ```
+
+    So, just run this command:
+
+    ```sh
+    sudo mount /dev/vdb /mnt -o spectator
```
Comment (trispera) [inline docs/cloud/pouta/multiattach.md:292]: Should we precise the type?
```
sudo mount /dev/vdb /mnt -t gfs2 -o spectator
```

**Comment by trispera** [comment]
LGTM. Just one comment

File: docs/cloud/pouta/multiattach.md (hunk 4 above)
Comment (lvarin) [reply docs/cloud/pouta/multiattach.md:291]: Thinking....

Comment (lvarin) [reply docs/cloud/pouta/multiattach.md:291]: Ok, let's add it, with a note that it is not necessary, but recommended to make sure the correct device is mounted.

What do you think?

Comment (trispera) [reply docs/cloud/pouta/multiattach.md:291]: Agree 👍

**Review by trispera — APPROVED** [review]
LGTM

**Pull request merged**